statistics/table.go

```diff
@@ -42,8 +43,9 @@ const (
 // Table represents statistics for a table.
 type Table struct {
 	Info    *model.TableInfo
-	Columns []*Column
-	Indices []*Column
+	m       sync.Mutex
```

Comment:
This lock is only used when build columns, and after that it is not used anymore.